integration/service/oasis/stationfinder/cloudfinder/low_energy_location_station_finder.go

```diff
@@ -29,12 +30,12 @@ func (sf *lowEnergyLocationStationFinder) prepare() {
 		searchcoordinate.Coordinate{
 			Lat: sf.location.Lat,
 			Lon: sf.location.Lon},
-		lowEnergyLocationCandidateNumber,
+		LowEnergyLocationCandidateNumber,
 		-1)
 	sf.bf.getNearbyChargeStations(req)
 	return
 }
 
-func (sf *lowEnergyLocationStationFinder) IterateNearbyStations() <-chan stationfindertype.ChargeStationInfo {
+func (sf *lowEnergyLocationStationFinder) IterateNearbyStations() <-chan *stationfindertype.ChargeStationInfo {
```

Comment:
Nit: use `l` instead of `sf` might be better as convention.

Reply:
Copy that, use `l` or meaningful name.